CONTRIBUTING.rst

```diff
@@ -107,7 +107,7 @@ At each release:
 
 Right after the release:
 
-- Update the version number to a ``x.y.dev`` number in
+- Update the version number to a ``x.y-dev`` number in
```

Comment:
The `.dev` notation is the recommandation for Python: https://www.python.org/dev/peps/pep-0440/#pre-releases

But semantic versionning (http://semver.org) requires `-dev`, and since PEP 440 allows `-` as an alias for `.`, why not go for it.

This should actually be x.y.z-dev.